.github/workflows/build.yaml

```diff
@@ -59,13 +55,9 @@ jobs:
           - openevse_esp32-poe-iso
           - openevse_esp32-heltec-wifi-lora-v2
         gui:
-          - name: gui-v1
-            repo: OpenEVSE/openevse_wifi_gui
-            build_flags: ""
-            ref: ${{ inputs.v1_ref }}
           - name: gui-v2
-            repo: KipK/openevse-gui-v2
-            build_flags: "-D DISABLE_WIFI_PORTAL -D WEB_SERVER_ROOT_PAGE_INDEX"
```

Comment:
These need defining in the C code

Reply:
Haven't I removed them on this pr ?

Reply:
ho it was on my other working branch. forgot to cherry pick it here

Reply:
I don't believe so, althopugh that is indeed the other option

Reply:
I've put it back should be good now